meteor/sofie-server/data-access/repositories/test/mongo-part-repository.spec.ts

```diff
@@ -135,23 +144,13 @@ describe(`${MongoPartRepository.name}`, () => {
 		})
 	})
 
-	// TODO: Extract to Helper Class in Model layer
-	function createPiece(params: { id?: string; name?: string; partId?: string }): Piece {
-		return new Piece({
-			id: params.id ?? testDatabase.getValidObjectIdString('id'),
-			name: params.name ?? 'name' + Math.random(),
-			partId: params.partId ?? 'segmentId' + Math.random(),
-		} as PieceInterface)
-	}
-
-	// TODO: Extract to Helper Class in Model layer
-	function createPart(params: { id?: string; name?: string; rank?: number; segmentId?: string }): Part {
-		return new Part({
-			id: params.id ?? testDatabase.getValidObjectIdString('id'),
-			name: params.name ?? 'name' + Math.random(),
-			rank: params.rank ?? Math.random(),
-			segmentId: params.segmentId ?? 'segmentId' + Math.random(),
-		} as PartInterface)
+	function createPart(mongoPartInterface?: Partial<MongoPart>): MongoPart {
```

Comment:
You are creating a `MongoPart`, not a `Part`, so the method should be named `createMongoPart`

Reply:
Has been renamed.